zk/README.md

```diff
@@ -31,6 +31,7 @@ To configure this check for an Agent running on a host:
 2. [Restart the Agent][6].
 
 #### Enabling SSL
+_Available for Agent versions >=7.24.0 and ZooKeeper versions >=2.9.0_
```

Comment:
zookeeper integration version or the zookeeper software version? Let's clarify that and link to changelog if applicable?